profile/README.md

```diff
@@ -7,25 +7,29 @@ computing by leveraging
 [Trusted Execution Environments](https://en.wikipedia.org/wiki/Trusted_execution_environment) to 
 protect containers and data.
 
-Our key considerations are:
+Goals:
 - Allow cloud native application owners to enforce application security requirements
 - Transparent deployment of unmodified containers
 - Support for multiple TEE and hardware platforms
 - A trust model which separates Cloud Service Providers (CSPs) from guest applications
 - Least privilege principles for the Kubernetes Cluster administration capabilities which impact 
 delivering Confidential Computing for guest application or data inside the TEE.
 
-### Find out more...
-- [Documentation](https://github.com/confidential-containers/documentation) : Find out more about 
-the progress to date, overall goals and work in progress.
+### Find out more
+- [**Documentation**](https://github.com/confidential-containers/documentation) Learn about our 
+vision, goals, and progress.
 
-### Get started quickly... 
-- [Kubernetes Operator for Confidential Computing](https://github.com/confidential-containers/confidential-containers-operator)
-: An operator to deploy confidential containers runtime (and required configs) on a Kubernetes 
-cluster
+### Get started
+- [**CC Kubernetes Operator**](https://github.com/confidential-containers/confidential-containers-operator)
```

Comment:
Not sure on CC as we haven't really highlighted what CC is short for?
However I wonder if we even need to have CC at all. Just say Kubernetes Operator, the description after makes it clear?